### ISSUE TITLE
Accessibility Issues

### ISSUE THREAD

**Issue by maryscar** [swagger-api/swagger-ui#4125]
| Q                               | A
| ------------------------------- | -------
| Bug or feature request?         |  Bug
| Which Swagger/OpenAPI version?  | Latest
| Which Swagger-UI version?       | 3.9.0
| How did you install Swagger-UI? | swagger-net
| Which browser & version?        | Microsoft Edge 40.15063.674.0, Chrome 63.0.3239.132a
| Which operating system?         | Windows 10 v1703

### Expected Behavior
1 ) Models span fails color contrast ratio thresholds for accessibility.
2 ) Lock icon fails to have discernible text. Primarily used by text readers.

### Current Behavior
1 ) 
![image](https://user-images.githubusercontent.com/34582625/35060237-abcab19e-fb72-11e7-926b-efc512486e26.png)
2 ) 
![image](https://user-images.githubusercontent.com/34582625/35060207-90a53d62-fb72-11e7-9d55-6e45f1c86455.png)

### Possible Solution
1 ) Increase the weight to the model link. 
$section-models-h4-font-color: $gray-600 !default;
2 ) Add an aria-label to the button. 

### Context
Accessibility testing was completed using internal tools as well as Google Accessibility Extension (https://chrome.google.com/webstore/detail/accessibility-developer-t/fpkknkljclfencbdbgkenhalefipecmb) and aXe Accessibility Engine (https://www.deque.com/products/axe/). These failures were the most consistent and effect users with visual accessibility issues. Will submit a pull request with fixes.

**Comment by maryscar** [comment]
Merged PR that fixes issue.
